x/ccv/provider/keeper/keeper.go

```diff
@@ -1066,3 +1075,17 @@ func (k Keeper) GetSlashLog(
 func (k Keeper) BondDenom(ctx sdk.Context) string {
 	return k.stakingKeeper.BondDenom(ctx)
 }
+
+// helper func
+func StrToConsumerConsAddresses(addresses []string) ([]types.ConsumerConsAddress, error) {
+	consAddresses := []types.ConsumerConsAddress{}
+	for _, address := range addresses {
+		// reverse of ConsumerConsAddress.String()
+		consAddr, err := sdk.ConsAddressFromBech32(address)
```

Comment:
if someone can help confirm`sdk.ConsAddressFromBech32()` is the reverse of `ConsumerConsAddress.String()` ?

Reply:
see how consumer handles this https://github.com/cosmos/interchain-security/blob/46251c9184eda7289688e08740dcebe03bd4a50a/x/ccv/consumer/keeper/relay.go#L81. Although I'm not sure that we need to add the complexity of converting these types back and forth so much